### PR TITLE
Fix result calculations

### PR DESCRIPTION
**Pull request by kellynm**
Switched to compute stems_inspected_completion within the inspection for loop instead of setting the value based on n_units_to_inspect. This is not the best solution, but may be ok for now. See issue #75 for explanation of best solution.

### CODE REVIEW

**Comment by kellynm** [comment]
@wenzeslaus This is ready to merge, except for the pylint disable line for inspect(). It is too long for one line, but breaking it into two lines did not work. What is the best way to handle this?

**Comment by wenzeslaus** [comment]
> @wenzeslaus This is ready to merge, except for the pylint disable line for inspect(). It is too long for one line, but breaking it into two lines did not work. What is the best way to handle this?

The fix for that is in #74. It's just about syntax. You can grab it from there and close #74.

File: tests/test_scenarios/config.yml [hunk 1]
```diff
@@ -39,14 +39,14 @@ pest:
...
-    - Actinidia
-    max_boxes: 10  # do not apply to shipments larger than
+# release_programs:
+#   naive_cfrp:
+#     flowers:
+#     - Hyacinthus
+#     - Gerbera
+#     - Rosa
+#     - Actinidia
+#     max_boxes: 10  # do not apply to shipments larger than
```
Comment (wenzeslaus) [inline tests/test_scenarios/config.yml:49]: You can just delete these lines.

File: pathways/inspections.py [hunk 2]
```diff
@@ -455,6 +472,8 @@ def inspect(config, shipment, n_units_to_inspect):
                 if ret.infested_stems_detection > 0:
                     # Update detected variable if infested stems found in box
                     detected = True
+                stems_inspected += inspect_per_box
+            assert ret.stems_inspected_completion == n_units_to_inspect
```
Comment (wenzeslaus) [inline pathways/inspections.py:476]: :+1: for the assert!

**Pull request merged**